client/src/styles/Stylesheet.tsx

```diff
@@ -1,114 +1,169 @@
+    }
+  },
+  {
+    selector: "node[type='trace']",
+    style: {
+      label: 'data(label)',
+      color: 'white',
+      'border-width': '1px',
+      'border-style': 'solid',
+      'border-color': '#A4C3D2',
+      'border-opacity': '1',
+      'font-family': 'SF Pro',
+      height: 50,
+      width: 50,
+      'background-color': '#A4C3D2',
+      'text-outline-color': 'black',
+      'text-outline-width': 1
+    }
   },
   {
     selector: "node[type='client']",
     style: {
+      label: 'data(label)',
+      color: 'white',
       'border-width': '1px',
-      'border-color': 'grey',
-      
+      'font-family': 'SF Pro',
+      'background-image': 'data:text/html;charset=utf-8' + encoded,
+      "background-width": ["100px", "100px", "100px"],
+      "background-height": ["100px", "100px", "100px"],
+      "background-image-smoothing": 'yes',
+      "background-fit": "contain",
       height: 50,
       width: 50,
-      shape: 'triangle',
-      'background-color': 'green',
-    },
+      'text-outline-color': 'black',
+      'text-outline-width': 1
+    }
   },
   {
     selector: "node[type='namespace']",
     style: {
+      label: 'data(label)',
+      color: 'white',
       'border-width': '1px',
       'border-color': 'grey',
-      
       height: 50,
       width: 50,
       shape: 'triangle',
-      'background-color': 'lightblue',
+      'background-color': '#FFFAA0',
+      'font-family': 'SF Pro',
+      'text-outline-color': 'black',
+      'text-outline-width': 1
     },
   },
   {
     selector: "node[type='root']",
     style: {
       'border-width': '1px',
       'border-color': 'grey',
-      
+      'font-family': 'SF Pro',
+      'text-outline-color': 'black',
+      'text-outline-width': 1,
       height: 50,
       width: 50,
       shape: 'rectangle',
-      'background-color': 'green',
+      'background-color': '#006994',
     },
   },
   {
     selector: 'edge',
     style: {
       width: 1,
-      // 'line-color': 'data(color)',
-      // 'target-arrow-color': '#ccc',
-      // 'target-arrow-shape': 'triangle',
       'curve-style': 'bezier'
     },
   },
   {
     selector: "edge[type='arrow']",
     style: {
       width: 1,
-      // 'line-color': 'data(color)',
-      'font-size': '5px',
+      'line-color': 'white',
+      'font-size': '13px',
       // 'target-arrow-color': '#ccc',
       'target-arrow-shape': 'triangle',
+      'target-arrow-color': 'white',
       'curve-style': 'bezier',
       'label': 'data(label)',
       'color': 'black',
-      'text-background-color': 'white'
+      'text-background-color': 'white',
+      'font-family': 'SF Pro',
     },
   },
   {
     selector: ".background",
     style: {
         "text-background-opacity": 1,
-        "color": "black",
-        "text-background-color": "whitesmoke",
+        "color": "white",
+        "text-background-color": "#161820",
+        'font-family': 'SF Pro',
     }
-  }
+  },
+  // {
```

Comment:
probably not needed

Reply:
yea, we can remove. The wrapping function is currently crashing cytoscape.